lookout/style/typos/templates/commits_with_typo_dataset_report.jinja2

```diff
@@ -7,11 +7,11 @@
                          "detection_false_negatives",
                          "fix_accuracy", "top3_fix_accuracy", "support", "review_time"] %}
 {{ tabulate(scores[scores_to_show].to_frame(),
-            tablefmt="pipe", headers=["metric", "value"], floatfmt=".3f",
-            stralign="right", numalign="left") }}
+            tablefmt="pipe", headers=["metric", "value"], floatfmt=".3g",
+            stralign="right", numalign="decimal") }}
```

Comment:
you can see in tests how it looks like now.
https://github.com/src-d/style-analyzer/pull/737/files#diff-0e070befae627e0df2e7108d23807696R172